main/src/ca/uwaterloo/flix/language/phase/ControlFlowAnalysis.scala

```diff
@@ -369,6 +369,19 @@ object ControlFlowAnalysis {
         // Unsoundly ignore the return value.
         AbstractValue.Bot
 
+      //TODO SJ: What to do?
```

Comment:
This is fine for now. You can add add a TODO:
AbstractValue.Bot // Add support for Channels to ControlFlowAnalysis.